src/neo-vm/ExecutionEngine.cs

```diff
@@ -843,6 +843,51 @@ private void ExecuteInstruction()
                         Push(x1 % x2);
                         break;
                     }
+                case OpCode.POW:
+                    {
+                        var exponent = (int)Pop().GetInteger();
+                        var value = Pop().GetInteger();
+                        Push(BigInteger.Pow(value, exponent));
+                        break;
+                    }
+                case OpCode.SQRT:
+                    {
+                        var value = Pop().GetInteger();
+
+                        if (value < 9)
```

Comment:
Why is this condition needed?

Reply:
SpeedUp, I can remove it

Reply:
mmm.. The result it's different, maybe we need a different formula